README.md

```diff
@@ -280,4 +280,12 @@ Benct Philip Jonsson
 
 # LICENSE
 
-GNU General Public License, Version 2
+This software distribution is subject to the EUPL, Version 1.2 or subsequent
```

Comment:
If ReadMeFromPod (or something similar) is used, this gets generated from the main module and is included as well. (it will probably show up in the diff after a build/release if you do copyfrombuild or copyfromrelease).